File: 백준/Bronze/34400. 민규의 서카디안 리듬/민규의 서카디안 리듬.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int t,n=scan.nextInt();
        for(int i=0;i<n;i++) {
            t=scan.nextInt();
            System.out.println(t%25<17?"ONLINE":"OFFLINE");
        }
    }
}

File: 백준/Bronze/34400. 민규의 서카디안 리듬/민규의 서카디안 리듬.c
#include <stdio.h>
int main(void) {
    int n,t;
    scanf("%d",&n);
    for(int i=0;i<n;i++) {
        scanf("%d",&t);
        printf("%s\n",t%25<17?"ONLINE":"OFFLINE");
    }
    return 0;
}